validate.py

```diff
@@ -63,11 +65,13 @@ def validate(event, context):
 
     tests = db['test']
 
+
     #try to find our user
-    results = tests.find_one({"email":email})
+    results = tests.find_one({"email":email}) #our collection is in a table organized as "metadata" : data (this is how json organizes data tables)
```

Comment:
The comment is a bit misleading to people who know about JSON. It's more literal to be elusive (`the data is stored in JSON`) or explain  a bit more (`email is the key to query by and Mongo takes in Dictionaries that it turns into JSON to execute the query`).

I generally don't want your notes to end up in the codebase because either people should figure it out or we should document more consistently. I favor the latter, where I link to the library documentation, because documenting how we use the library (in, generally, the most obvious way) should be left to the library's documentation.

Reply:
Yeah I had deleted the notes but I think something or another happened with git (this was my first pull) and I may have re-updated my own copy before I ended up sending over the pull my bad.